spinalcordtoolbox/process_seg.py

```diff
@@ -29,6 +29,22 @@
 # NB: We use a threshold to check if an array is empty, instead of checking if it's exactly 0. This is because
 # resampling can change 0 -> ~0 (e.g. 1e-16). See: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3402
 NEAR_ZERO_THRESHOLD = 1e-6
+KEYS_DEFAULT = ['area',
+                'angle_AP',
+                'angle_RL',
+                'diameter_AP',
+                'diameter_AP_ellipse',
+                'diameter_RL',
+                'eccentricity',
+                'orientation',
+                'solidity',
+                'length'
+                ]
+KEYS_HOG = ['centermass_x', 'centermass_y', 'angle_hog']
+KEYS_QUADRANT = ['area_quadrant_anterior_left', 'area_quadrant_anterior_right', 'area_quadrant_posterior_left',
+                 'area_quadrant_posterior_right']
+KEYS_SYMMETRY = ['symmetry_dice_RL', 'symmetry_hausdorff_RL', 'symmetry_difference_RL', 'symmetry_dice_AP',
+                 'symmetry_hausdorff_AP', 'symmetry_difference_AP']
```

Comment:
Tiny tiny nit (because it makes it easier to see the symmetry between the two lines)

```suggestion
KEYS_QUADRANT = ['area_quadrant_anterior_left', 'area_quadrant_anterior_right', 
                 'area_quadrant_posterior_left', 'area_quadrant_posterior_right']
KEYS_SYMMETRY = ['symmetry_dice_RL', 'symmetry_hausdorff_RL', 'symmetry_difference_RL',
                 'symmetry_dice_AP', 'symmetry_hausdorff_AP', 'symmetry_difference_AP']
```